packages/core/src/common/keybinding.ts

```diff
@@ -170,7 +237,7 @@ export class KeybindingRegistry {
      * @param commandId The ID of the command for which we are looking for keybindings.
      */
     getKeybindingsForCommand(commandId: string): Keybinding[] {
-        return this.commands[commandId] || [];
+        return this.keymapsIndex.getKeybindingsForCommand(commandId) || [];
```

Comment:
I think this is weird... we should merge the keymapIndex into the keybinding registry.. there's no need IMO to have 2 entities .. the KeybindingRegistry should be smart enough on it's own to understand scopes.